### PR TITLE
Sensor refactor 1.0

### PR DESCRIPTION
**Pull request by jrreyna**
## Motivation and Context

For habitat-sim PR #1932 https://github.com/facebookresearch/habitat-sim/pull/1932. The sensor refactor in sim requires a few updates on the lab side, e.g., simulator.py in habitat sim no longer uses the simulator.Sensor wrapper class for sensor.Sensor. Now it just uses sensor.Sensors directly. This requires removing that one layer of abstraction from anywhere referenced in habitat-lab

## How Has This Been Tested

locally
## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: habitat-lab/requirements.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@ pickle5; python_version < '3.8'
 # visualization optional dependencies
 imageio>=2.2.0
 imageio-ffmpeg>=0.2.0
+pre-commit
```
Comment (Skylion007) [inline habitat-lab/requirements.txt:11]: No reason this should be in requirements?

Comment (jrreyna) [reply habitat-lab/requirements.txt:11]: Yeah my bad, I was looking up how to install pre-commit, and the website I was looking at said to add it to the requirements.txt file. Wasn't exactly sure if it was necessary

File: .circleci/config.yml [hunk 2]
```diff
@@ -193,6 +193,8 @@ jobs:
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat;
               cd habitat-sim
+              git fetch --all
+              git checkout sensor_refactor_python
```
Comment (0mdc) [inline .circleci/config.yml:197]: Make sure to discard this before merging.

**Review by 0mdc — APPROVED** [review]
LGTM!

**Pull request closed without merging**